packages/main/src/plugin/acp/acp-session-manager.ts

```
@@ -0,0 +1,964 @@
+    }
+
+    console.log(`[ACP] initializing connection...`);
+    const initResult = await session.connection.initialize({
+      protocolVersion: acp.PROTOCOL_VERSION,
+      clientCapabilities: {
+        fs: { readTextFile: true, writeTextFile: true },
+      },
+    });
+    console.log(`[ACP] initialized: protocol v${initResult.protocolVersion}`);
+
+    console.log(`[ACP] creating new session...`);
```

Biome (2.4.16)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)